backend/core/models.py

```diff
@@ -1680,6 +1680,23 @@ class RiskScenario(NameDescriptionMixin):
         ("transfer", _("Transfer")),
     ]
 
+    QUALIFICATIONS = [
+        ("Financial", _("Financial")),
+        ("Legal", _("Legal")),
+        ("Reputation", _("Reputation")),
+        ("Operational", _("Operational")),
+        ("Confidentiality", _("Confidentiality")),
+        ("Integrity", _("Integrity")),
+        ("Availability", _("Availability"))
+    ]
```

Comment:
We should allow impact qualifications to be specified in risk matrices, as it is done for strength of knowledge. Most likely some users will want to have their own

Reply:
The default values can be what is proposed here.